src/ddmd/parse.d

```diff
@@ -261,6 +259,7 @@ final class Parser : Lexer
     Loc endloc; // set to location of last right curly
     int inBrackets; // inside [] of array index or slice
     Loc lookingForElse; // location of lonely if looking for an else
+    ASTFamily astFam;
```

Comment:
No need for a member - just use the static types. If methods are needed in some ASTFamily, they can be defined static.

In the future if we have an AST family keeping its own state (unlikely), we may go with a member.